### ISSUE TITLE
Sorting a store's listings by Date (newest) and Date (oldest) would be useful

### ISSUE THREAD

**Issue by morebrownies** [OpenBazaar/openbazaar-desktop#217]
I just thought of this sort option. It would be nice to add it in at some point. 

<img width="818" alt="screenshot 2016-12-15 13 02 12" src="https://cloud.githubusercontent.com/assets/13066886/21241930/c2898d50-c2c6-11e6-8a96-3be71551d1ec.png">

**Comment by rmisio** [comment]
To do this we need a corresponding issue on the server for a 'created date' to come back in the listing API.

**Comment by drwasho** [comment]
Good suggestion @morebrownies

**Comment by cactux** [comment]
Good idea. Would it be done on the creation date or update date? Or both?
